pymeasure/adapters/telnet.py

```diff
@@ -46,8 +46,8 @@ def __init__(self, host, port=0, query_delay=0, **kwargs):
         for kw in kwargs:
             if kw not in safe_keywords:
                 raise TypeError(
-                    f"TelnetAdapter: unexpected keyword argument '{kw}', "
```

Comment:
If you want you can bring the f-strings back, we just removed Python3.5 support yesterday.